src/viur/core/modules/user.py

```diff
@@ -80,7 +80,6 @@ class UserSkel(skeleton.Skeleton):
 
     # Properties required by custom auth
     password = PasswordBone(
-        descr="Password",
         required=False,
```

Comment:
I think we could remove some default arguments. But that's not a big problem either.
```suggestion
```